kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/registry/DefaultLocalRegistryStorage.kt

```diff
@@ -0,0 +1,34 @@
+package com.kotlindiscord.kord.extensions.registry
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.map
+
+/**
+ * Default "local" implementation of [RegistryStorage] which internally
+ * uses a mutableMap.
+ */
+public class DefaultLocalRegistryStorage<K, T> : RegistryStorage<K, T> {
```

Comment:
Make it `open`, make things `public`. You know the drill. I'd stick a `@suppress` in the `registry` kdoc, though.